lib/engine/step/g_18_co/company_pending_par.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
```

Comment:
This file is not yet used. It is intended to contain the implementation of the "par groups" as certain corporations can only be parred in certain sections of the par choices.